### ISSUE TITLE
Recoil selector throws error in production build

### ISSUE THREAD

**Issue by kiran-venugopal** [facebookexperimental/Recoil#160]
**Recoil selector throwing an error of "Error: 's'  is read-only" in the production build of React js   ##app**

Here is the console error and code: 
<img width="759" alt="Screenshot 2020-05-21 at 1 10 53 PM" src="https://user-images.githubusercontent.com/62516337/82802865-3be2e100-9e9d-11ea-9a1a-f4b2d3a93f26.png">
<img width="608" alt="Screenshot 2020-05-21 at 1 11 05 PM" src="https://user-images.githubusercontent.com/62516337/82802928-5026de00-9e9d-11ea-9e74-98f8c82e9c33.png">

**Comment by Rafal-Jenczelewski** [comment]
This is already fixed by #69 and is a duplicate of #122.

We need all to wait for a release containing the fix.

**Comment by davidmccabe** [comment]
Just waiting on one more diff to fix our repo sync issues and then we can publish those fixes. Thanks for your patience!

**Comment by ShanonJackson** [comment]
New version asap as many things have been fixed in source.

**Comment by hagi4u** [comment]
I can't build to production because same issue.

**Comment by Rafal-Jenczelewski** [comment]
I think this is fixed as version 0.0.8 was released?

**Comment by ma-pe** [comment]
I can confirm this i fixed with 0.0.8 🚀

**Comment by ShanonJackson** [comment]
Thought i'd just leave this comment here even though its not the place for it; After refactoring a small project from redux/thunk -> recoil; We saw a 25-35% reduction in page size, and 15KB or so wiped off our "all components share this code" bundle. We refactored purely as an experiment without knowing if it would be merged in or not but the results blew us away and now everything will stay recoil going forwards.
